templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -236,9 +236,9 @@ echo ::endgroup::
 if [[ "$TEST" = "azure" ]]; then
   {%- if pulp_scheme == "https" %}
   AZCERTIFI=$(/opt/az/bin/python3 -c 'import certifi; print(certifi.where())')
-  cat /usr/local/share/ca-certificates/azcert.crt >> $AZCERTIFI
-  cat /usr/local/share/ca-certificates/azcert.crt | cmd_stdin_prefix tee -a /usr/local/lib/python3.8/site-packages/certifi/cacert.pem > /dev/null
+  cat /usr/local/share/ca-certificates/azcert.crt | cmd_stdin_prefix tee -a $(AZCERTIFI) > /dev/null
```

Comment:
AZCERTIFI seems to be the path that we wanted here.  That is, the path to the `/usr/local/lib/python3.8/site-packages/certifi/cacert.pem` where the location is correct.

Reply:
It doesn't look like you properly converted the commands. If `certifi.where() = .../certifi/cacert.pem` then the old commands do:

1. Append `azcert.crt` to `certifi/cacert.pem`
2. Write `azcert.crt` to `certific/cacert.pem` (Seems we just override step 1 if `AZCERTIFI` is equal to `python3.8/.../certifi/cacert.pem`)
3. Write `azcert.crt` to `pki/tls/cert.pem`

The new commands do:

1. Write `azcert.crt` to `certifi/cacert.pem` 
2. Write `azcert.crt` to `pki/tls/cert.pem`
3. Append `azcert.crt` to `certifi/cacert.pem` (Now this file has double copies of `azcert.crt`

The outputs of the new commands are not the same.

Reply:
My analysis was wrong, I failed to notice the `cmd_stdin_prefix` which routes the command to the container. Seeing as the pulpcore CI is still failing, the next question to ask is does `/opt/az/bin/python3` exist inside the pulp ci container? It doesn't in the oci_env, so I am suspicious that it does in the ci container.